src/index.js

```diff
@@ -40,3 +40,82 @@ const siteContent = { // DO NOT CHANGE THIS OBJECT
 };
 
 console.log('project wired!')
+
+// Declaring Variables
+const nav = document.querySelectorAll('nav')
+const navLinks = document.querySelectorAll('nav a')
+const navServices = document.querySelectorAll('nav a')[0]
```

Comment:
Instead of making these variables that you only use once to inject the text you could do it this way

```
const aLinkArray = Object.values(siteContent.nav);
navLinks.forEach(function(element, index){
  element.textContent = aLinkArray[index];
})
```

This injects the text into the nav links without having the declare them as variables